src/components/Common/Input/Input.tsx

```diff
@@ -13,10 +13,17 @@ export interface InputProps {
 
 const Input = ({ name, value, onChange, label, placeholder, error, className, type = 'text' }: InputProps) => {
   return (
-    <div>
-      {label && <div>{label}</div>}
+    <div className={'input-container'}>
+      {label && <div className={'input-label'}>{label}</div>}
```

Comment:
For accessibility purposes it would be a good idea to use the `<label>` tag to show the label and use the `htmlFor` attribute with the input's id